src/NServiceBus.StructureMap/NServiceBus.StructureMap.csproj

```diff
@@ -12,8 +12,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="[7.0.0-*,8.0.0)" />
     <PackageReference Include="StructureMap" Version="[4.3.0,5.0.0)" />
+    <PackageReference Include="NServiceBus" Version="[7.0.0-beta0009,8.0.0)" />
```

Comment:
Is this intentional?

Reply:
I assume so @mauroservienti, the diagnostics are only from beta 9 updwards

Reply:
this is redundant. the build server would have produced that dependency range on the next build.